rfcs/trust-and-safety/20230109-trust_and_safety_overview.md

```diff
@@ -0,0 +1,266 @@
+# Trust and Safety in the Openverse: preliminary overview and exploration
+
+This document seeks to explore the facets of Trust and Safety that Openverse
+needs to consider as we undertake the important work of building systems and
+tools for content moderation. The bulk of this practice is a list of user
+stories and accompanying assumptions and technical and process requirements
+needed to meet the assumptions.
+
+## General considerations
+
+### Accessibility
+
+While considering the user stories and accompanying requirements and
+assumptions, keep in mind that content moderation is not just about keeping
+Openverse legal. It is also about accessibility. If Openverse users are
+confronted with sensitive materials without their consent, then Openverse is not
+an accessible platform. Likewise, Openverse is used by a diverse set of people,
+including children in educational environments. Consider that the age and other
+intersectionalities of a user will influence how they're affected by content
+moderation policies (both successes and failures).
+
+### The relevance of scanning for known illegal materials
+
+Note that a distinct feature of Openverse _as it exists today, in early 2023_ is
+that there is no method for anyone to upload content directly to us. Every
+single provider is manually added to the catalogue. We currently rely heavily on
+the fact that our content providers have their own content moderation policies
+and procedures. The majority of our content providers (to the best of our
+knowledge) that have user generated content automatically scan that content for
+illegal materials. The other providers are either unlikely to ever include
+illegal materials (GLAM institutions) or manually review each submission by hand
+(WordPress Photo Directory). This may not always be the case. One of the future
+goals for Openverse is for websites (and WordPress sites, in particular) to be
+able to automatically add themselves as providers of openly licensed content.
+Once that is possible, then it will become even more imperative that Openverse
+scans materials for illegal materials. Even before then, however, I think it is
+prudent for us to consider doing it earlier because running the risk of hosting
```

Comment:
Tiny nit: we are not really "hosting" the media. We host the metadata and cache the thumbnails, but the media hosting itself is still done by the providers.

Reply:
That's a good qualification to make, though I'm not sure how to do it without making the qualification over and over again. Do you have a handy turn of phrase that could be used here to indicate the extent to which we _do_ host some aspects of the content (textual and visual, when it comes to thumbnails)?

Reply:
The word "present" seems to be valid in our case. We don't "host" the content but we "surface" and "showcase" the content in our web space.

Reply:
Fair enough. I still think there's an important technical distinction to draw that we have extra obligations to ensure that our independently hosted versions (cached thumbnails) of results are correctly managed, but I'll change the language to be "present" with a separate qualification elsewhere.

Reply:
Changes made in https://github.com/WordPress/openverse/pull/362/commits/d63707395a9007415047bfe4a1a99efab5490ec3